examples/virus_on_network_directed/virus_on_network_directed/model.py

```diff
@@ -0,0 +1,163 @@
+import math
+from enum import Enum
+import networkx as nx
+
+from mesa import Agent, Model
+from mesa.time import RandomActivation
+from mesa.datacollection import DataCollector
+from mesa.space import NetworkGrid
+
+
+class State(Enum):
+    SUSCEPTIBLE = 0
+    INFECTED = 1
+    RESISTANT = 2
+
+
+def number_state(model, state):
+    return sum(1 for a in model.grid.get_all_cell_contents() if a.state is state)
+
+
+def number_infected(model):
+    return number_state(model, State.INFECTED)
+
+
+def number_susceptible(model):
+    return number_state(model, State.SUSCEPTIBLE)
+
+
+def number_resistant(model):
+    return number_state(model, State.RESISTANT)
+
+
+class VirusOnNetwork(Model):
```

Comment:
Needs to be `VirusOnNetworkDirected`.

Reply:
Reminder to rename all occurrences of `VirusOnNetwork` inside examples/virus_on_network_directed/.

Reply:
All occurrences should now be renamed